pandas.errors.OutOfBoundsDatetime
=================================

.. currentmodule:: pandas.errors

.. autoexception:: OutOfBoundsDatetime